pkgs/by-name/co/cosmic-player/package.nix

```
@@ -0,0 +1,96 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, makeBinaryWrapper
+, alsa-lib
+, cosmic-icons
+, ffmpeg
+, llvmPackages
+, just
+, pkg-config
+, libxkbcommon
+, wayland
+, xorg
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cosmic-player";
+  version = "unstable-2024-06-10";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = pname;
```

```suggestion
    repo = "cosmic-player";
```
[Avoid `repo = pname`](https://github.com/nix-community/nixpkgs-lint/issues/21)